java/pom.xml

```diff
@@ -1070,6 +1070,8 @@
                     -DCMAKE_INSTALL_LIBDIR=lib/${os.detected.arch}
                     -DCMAKE_INSTALL_PREFIX=${arrow.dataset.jni.dist.dir}
                     -DCMAKE_PREFIX_PATH=${project.basedir}/../java-dist/lib/${os.detected.arch}/cmake
+                    -DProtobuf_USE_STATIC_LIBS=ON
+                    -DProtobuf_ROOT=${project.basedir}/../cpp-jni/protobuf_ep-install
```

Comment:
Just out of curiosity, why are we making this change?

Reply:
Ah, is this for https://github.com/apache/arrow/blob/main/java/gandiva/CMakeLists.txt#L41 ?

We can use this solution for now but we should find a better solution that doesn't require additional CMake options such as `Protobuf_ROOT` and `Protobuf_USE_STATIC_LIBS`. Could you open an issue for it?
(We may add support for installing bundled `protoc` for this case.)